CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ description: |-
 # VKCS Provider's changelog
 
 #### v0.1.13 (unreleased)
+- Added retry of creating resource_vkcs_images_image with default store if initial creation failed
```

Comment:
Allow create images in clouds w/o s3 support.

Reply:
Rewritten